shared/src/commonMain/kotlin/co/nimblehq/kmm/showcase/data/Api.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.kmm.showcase.data
+
+import io.github.aakira.napier.Napier
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.features.json.serializer.*
+import io.ktor.client.features.logging.*
+import kotlinx.serialization.json.Json
+
+expect fun httpClient(config: HttpClientConfig<*>.() -> Unit): HttpClient
```

Comment:
I don't see its usage now, do we need this?

Reply:
@luongvo It's the specific config for overriding on each platform if needed. I think we could keep it for now until we do integration on each platform.

Reply:
okay, look forward to see our further config here